Source/Scene/ModelExperimental/PickingPipelineStage.js

```diff
@@ -66,12 +68,60 @@ PickingPipelineStage.process = function (
   }
 };
 
-function processInstancedPickIds(renderResources, context) {
+function processPickTexture(renderResources, primitive, instances) {
+  var model = renderResources.model;
+  var featureIdAttribute;
+  var featureIdAttributeIndex = model.featureIdAttributeIndex;
+
+  if (defined(instances)) {
+    featureIdAttribute = instances.featureIdAttributes[featureIdAttributeIndex];
+  } else if (primitive.featureIdTextures.length > 0) {
+    var featureIdTextureIndex = model.featureIdTextureIndex;
+    featureIdAttribute = primitive.featureIdTextures[featureIdTextureIndex];
```

Comment:
To avoid the ambiguity of storing a feature id texture in a variable called `featureIdAttribute` maybe just have a variable for the feature table id.